### ISSUE TITLE
Publish ns1.ns1 as collection

### ISSUE THREAD

**Issue by ignatenkobrain** [ns1/ns1-ansible-modules#32]
I see that ns1.ns1 is now published as a role on galaxy, but this does not make sense from my POV and should be collection instead.

Would appreciate any comments.

**Comment by felixfontein** [comment]
I fully agree. Making this a collection also makes it a lot easier to both install and use the modules. The main disadvantage is that the minimally required Ansible version would be 2.9 (maybe 2.8 would also work, but collection support there is more a tech preview than something that is mature).

**Comment by felixfontein** [comment]
@mburtless does NS1 have any interest in publishing ns1.ns1 as a collection?

**Comment by pwaldon** [comment]
Bump in 2024
